data/recData/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  width: 400px;
+  height: 320px; }
+
+h1 {
+  font-size: 1em;
+  font-weight: normal; }
+
+.addon-box {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 30%;
+  width: 400px;
+  border-top: gray solid 1px; }
+  .addon-box div {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: center; }
+  .addon-box h2, .addon-box h3 {
+    width: 100%;
+    font-weight: normal;
+    margin: auto; }
+  .addon-box .image-box {
+    height: 100%;
+    width: 20%; }
+    .addon-box .image-box img {
+      display: block;
+      margin: auto; }
+  .addon-box .inner-box-1 {
+    height: 100%;
+    width: 60%; }
+  .addon-box .inner-box-2 {
+    margin: 0;
+    height: 100%;
+    width: 15%; }
+    .addon-box .inner-box-2 a {
+      width: 100%;
+      padding: auto;
+      text-decoration: none; }
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch button {display:none;}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 34px;
+  background-color: #ccc;
+  -webkit-transition: .4s;
```

Comment:
we shouldn't need vendor prefixed rules at all. if there is something that doesn't work without a prefix i suggest adding autoprefixer to the webpack build process.